cloud_build/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+token=$(gcloud auth application-default print-access-token)
```

Comment:
please confirm this can't be in any way run/viewed through the website/browser interface

Reply:
Yes, this script only runs in the build environment, so once build completes the VM or container used to build is destroyed and this variable is gone.  Further this variable "token" is not even exported, so the accessibility is limited to the shell.  I am pretty certain it won't be leaked.